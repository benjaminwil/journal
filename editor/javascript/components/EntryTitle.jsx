import React, {Component} from 'react';

class EntryTitle extends React.Component {
  render() {
    return (
      <h1 className="title">{this.props.title}</h1>
    )
  }
}

export default EntryTitle;
