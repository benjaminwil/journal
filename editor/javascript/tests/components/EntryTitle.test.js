import React from 'react';
import renderer from 'react-test-renderer';
import EntryTitle from '../../components/EntryTitle.jsx';

test('the EntryTitle component displays en entry title', () => {
  const props = {
      title: "I walked for a while",
    }

  const component = renderer
    .create( <EntryTitle title={props.title} />)
    .toJSON();

  expect(component).toMatchSnapshot();
});
