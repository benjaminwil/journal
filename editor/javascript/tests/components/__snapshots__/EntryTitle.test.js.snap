// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the EntryTitle component displays en entry title 1`] = `
<h1
  className="title"
>
  I walked for a while
</h1>
`;
